src/System Application/App/URI/src/UriBuilderImpl.Codeunit.al

```diff
@@ -219,7 +229,10 @@ codeunit 3062 "Uri Builder Impl."
         foreach CurrentKey in KeysWithValueList.Keys() do begin
             KeysWithValueList.Get(CurrentKey, CurrentValues);
             foreach CurrentValue in CurrentValues do
-                FinalQuery += '&' + Uri.EscapeDataString(CurrentKey) + '=' + Uri.EscapeDataString(CurrentValue);
+                if UseODataEncoding then
```

Comment:
I'd get rid of this if my moving it down into the helper :-)

Reply:
Moved down!